### DISCUSSION TITLE
[QUESTION] How can we take photos while using the bot?

### DISCUSSION THREAD

**Discussion by Scolymus** [python-telegram-bot/python-telegram-bot#3064] [Q&A] [answered]
### Issue I am facing

I want to send an image to a server, but I want to let the user to either chose the image from its files (I guess send_photo command) or taking it in the moment. How can we do the last? Any external library to use? Is it OS specific? 

### Operating System

Working under Mac M1

### Version of Python, python-telegram-bot & dependencies

```shell
python-telegram-bot 20.0a0
Bot API 6.0
Python 3.9.12 (main, Apr  5 2022, 06:56:58)  [GCC 7.5.0]
```

**Comment by Bibo-Joshi** [comment]
I don't understand what you're trying to do/where exactly you're stuck. Send an image to which server? Please have a look at [this guide](../wiki/Ask-Right) on how to ask good questions.

**Reply by Scolymus** [reply]
I want to open the camera with a button in the bot if it is possible. Otherwise I will tell the user to click on the clip where you can attach files where the camera option is.

**Reply by clot27** [reply]
> I want to open the camera with a button in the bot if it is possible. Otherwise I will tell the user to click on the clip where you can attach files where the camera option is.

You can't do that, see https://t.me/pythontelegrambotgroup/593164

**Comment by Scolymus** [comment, marked as answer]
ok! got it. Thank you!
